### DISCUSSION TITLE
web gui not showing "scanning" when no traffic. It just stays on last call TG.

### DISCUSSION THREAD

**Discussion by frazt** [chuot/rdio-scanner#361] [Q&A]
beginner here and it seems to have come together ok. However on the web interface is isn't showing "scanning" while waiting for calls.  It simply sits on the last call TG shown until another call takes its place.  Any ideas?  Audio passes fine from one talk group to the next. This issue is purely visual.

**Comment by Clark985** [comment]
That's normal behavior I believe.  I see that with my install (and I'm fine with it).

**Reply by AvalancheAddict** [reply]
Correct, that is normal behavior.
